src/components/veNft/VeNftHeaderSection.tsx

```diff
@@ -0,0 +1,31 @@
+import { Trans } from '@lingui/macro'
+import React from 'react'
+
+interface VeNftHeaderSectionProps {
+  tokenName: string | undefined
+  tokenSymbolDisplayText: string
+  projectName: string
+}
+
+const VeNftHeaderSection = ({
+  tokenName,
+  tokenSymbolDisplayText,
+  projectName,
+}: VeNftHeaderSectionProps) => {
+  return (
+    <>
+      <h1>
+        <Trans>Lock {tokenName} for Voting Power</Trans>
```

Comment:
**question (non-blocking)**: wondering if `tokenSymbolDisplayText` might be more suitable here. No big though